oauth2-server/src/main/java/com/clouway/oauth2/IssueNewTokenActivity.java

```diff
@@ -57,14 +57,21 @@ public Response execute(Client client, Identity identity, Set<String> scopes, Re
       Pem.Block key = certificates.get(certKey);
       String host = request.header("Host");
 
+      Map<String, Object> claims = new LinkedHashMap<>();
+      claims.put("iss", host);
+      claims.put("aud", client.id);
+      claims.put("sub", identity.id());
+      claims.put("iat", instant.asDate());
+      claims.put("exp", new Date(accessToken.expirationTimestamp()));
```

Comment:
This is a snippet from master:

```java
return new BearerTokenResponse(accessToken.value, accessToken.ttlSeconds(instant), response.refreshToken, idToken);
```

exp in token and exp in id_token should be same, so and the code should be same.